Doc/c-api/type.rst

```diff
@@ -50,6 +50,22 @@ Type Objects
       The return type is now ``unsigned long`` rather than ``long``.
 
 
+.. c:function:: PyObject* PyType_GetDict(PyTypeObject* type)
+
+   Return the type object's internal namespace, which is otherwise only
+   exposed via a read-only proxy (``cls.__dict__``).  This is a
+   replacement for accessing :c:member:`~PyTypeObject.tp_dict` directly.
+   The returned dictionary must be treated as read-only.
+
+   This function isn't intended for general use.  It's meant for
```

Comment:
If it's not meant for general use, would it be better to add it as unstable API?

Reply:
To be honest, I think it makes more sense to say this _is_ for general use.